e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -191,16 +191,13 @@ describeEE("formatting > whitelabel", () => {
           );
           cy.findByText(
             "The image you chose is corrupted. Please choose another one.",
-          ).should("be.visible");
+          )
+            .scrollIntoView()
+            .should("be.visible");
           cy.findByText("corrupted-file.jpg").should("not.exist");
         });
 
-        cy.log("test removing the custom illustration");
-        /**
-         * I opted for uploading the same file twice rather than uploading once,
-         * testing the login page, and revisiting the settings page to test removing it.
-         * Since visiting multiple pages should make the test runs slower.
-         */
+        cy.log('test replacing the "corrupted" file with a valid one');
```

Comment:
This is basically the new workflow, previously we needed to select the `Custom` option again, but we couldn't do that on Cypress, we needed to click the hidden input file directly.

Now we can only click the hidden input file when the select is set to Custom. Which it's already on Custom from the previous step, so the test doesn't look like it changed, but the workflow acutally changed.